apollo-core/src/test/java/com/ctrip/framework/apollo/core/utils/StringUtilsTest.java

```diff
@@ -0,0 +1,94 @@
+package com.ctrip.framework.apollo.core.utils;
+
+import com.ctrip.framework.apollo.core.utils.StringUtils;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+public class StringUtilsTest {
+
+  @Test
+  public void testEqualsIgnoreCase() {
+    Assert.assertFalse(StringUtils.equalsIgnoreCase(",", "foo"));
+    Assert.assertFalse(StringUtils.equalsIgnoreCase(null, "??"));
+    Assert.assertTrue(StringUtils.equalsIgnoreCase(null, null));
+  }
+
+  @Test
+  public void testEquals() {
+    Assert.assertFalse(StringUtils.equals(null, ""));
+    Assert.assertTrue(StringUtils.equals(null, null));
+    Assert.assertTrue(StringUtils.equals("3", "3"));
+  }
+
+  @Test
+  public void testIsBlank() {
+    Assert.assertFalse(StringUtils.isBlank("\'"));
+    Assert.assertTrue(StringUtils.isBlank(""));
+  }
+
+  @Test
+  public void testIsContainEmpty() {
+    Assert.assertFalse(StringUtils.isContainEmpty(new String[] {}));
+    Assert.assertFalse(StringUtils.isContainEmpty(new String[] {"1"}));
+    Assert.assertTrue(StringUtils.isContainEmpty(new String[] {null}));
+  }
+
+  @Test
+  public void testIsEmpty() {
+    Assert.assertFalse(StringUtils.isEmpty("1"));
+    Assert.assertTrue(StringUtils.isEmpty(null));
```

Comment:
Another potential `true` test to cover all branches would be with an input of `""`